.nvmrc

```diff
@@ -0,0 +1 @@
+11.2
```

Comment:
Is this handy? Isn’t it nice that people are on different Node versions, means more configurations used, so less bugs?!

Reply:
I did this only so that `npm install`s don't lead to an updated `package-lock.json` due to variance in `npm` version. But given the conversation below around the lock file, I am down to remove this if we remove the lock file as well. It's predicated on what we decide below.

Reply:
Update: this is now nuked. ^_^